File: ExampleNodes/Android/YodiwoNode/app/src/main/java/com/yodiwo/plegma/StateDescription.java
package com.yodiwo.plegma;

/**
 * Created by ApiGenerator Tool (Java) on 3/8/2015 10:26:17 &#956;&#956;.
 */

public class StateDescription {

    public double Minimum;

    public double Maximum;

    public double Step;

    public String Pattern;

    public Boolean ReadOnly;

    public StateDescription() {
    }

    public StateDescription(double Minimum, double Maximum, double Step, String Pattern, Boolean ReadOnly) {
        this.Minimum = Minimum;
        this.Maximum = Maximum;
        this.Step = Step;
        this.Pattern = Pattern;
        this.ReadOnly = ReadOnly;

    }

}

File: Csharp/Yodiwo.Gateway/Openhab/Rest/Model/StateDescription.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Yodiwo.Gateway.Openhab.Rest.Model
{
    [Serializable]
    public class StateDescription
    {
        public double minimum { get; set; }
        public double maximum { get; set; }
        public double step { get; set; }
        public string pattern { get; set; }
        public bool readOnly { get; set; }
        public List<StateOption> options { get; set; }

        public StateDescription()
        {
            this.options = new List<StateOption>();
        }
    }
}
